atlassian_backup-1.0.0/PKG-INFO

```diff
@@ -0,0 +1,11 @@
+Metadata-Version: 1.2
+Name: atlassian_backup
+Version: 1.0.0
+Summary: Backup Atlassian Using Elastio
+Home-page: UNKNOWN
+Author: Vihar Chokshi
+Author-email: vc@iot4.net
+License: UNKNOWN
```

Comment:
Let's release this under MIT and Apache 2, unless the `confluence` and `jira` modules have an incompatible license in which case we have to use their license instead.